addons/web/controllers/main.py

```diff
@@ -2033,13 +2031,14 @@ def report_barcode(self, type, value, width=600, height=100, humanreadable=0, qu
         return request.make_response(barcode, headers=[('Content-Type', 'image/png')])
 
     @http.route(['/report/download'], type='http', auth="user")
-    def report_download(self, data, token, context=None):
+    def report_download(self, data, context=None):
```

Comment:
Is just removing the `token` parameter going to work since the function still passes it in? `report` can remove it because it already has a `**kw` but for this one that seems less likely.

Reply:
I already removed the line that add this param 'token' in get_file.

```diff
@@ -257,7 +248,6 @@ function get_file(options) {
             data.append(k, v);
         });
     }
-    data.append('token', 'dummy-because-api-expects-one');
     if (core.csrf_token) {
```

From what I see (grep in code + test of download report in backend, it was the only place to use it.
But if you prefer I can add it like for others controllers.

PS: `response_wrap` already catch the case where extra param will be provided to a controller, and they will be still available via request.params is really needed.

Reply:
Ok.